staubli_rx160_support/urdf/rx160_macro.xacro

```diff
@@ -18,6 +26,12 @@
           <mesh filename="package://staubli_rx160_support/meshes/rx160/collision/base_link.stl"/>
         </geometry>
       </collision>
+      <inertial>
+          <mass value="${base_mass}"/>
+          <inertia ixx="0.001" ixy="0.0" ixz="0.0"
+                   iyy="0.001" iyz="0.0"
+                   izz="0.001"/>
+        </inertial>
```

Comment:
We'll have to add a note to the package manifest telling users that the inertia elements in the xacro are 'fictional'' (like in [kuka_experimental/kuka_kr16_support/package.xml](https://github.com/ros-industrial/kuka_experimental/blob/11d8a061b85431e6647f949486858f1dddd9cb59/kuka_kr16_support/package.xml#L28)).